### PR TITLE
Use assignment instead of annotation in third party enums

### PR DESCRIPTION
**Pull request by hamdanal**
Follow up to #11956.

This PR handles all third party stubs except Pillow and pygit2 that I'll change in a follow up.

### CODE REVIEW

File: stubs/regex/regex/_regex_core.pyi [hunk 1]
```diff
@@ -44,42 +44,42 @@ class RegexFlag(enum.IntFlag):
...
-UNICODE: int
-V0: int
-VERSION0: int
-V1: int
-VERSION1: int
-W: int
-WORD: int
-X: int
-VERBOSE: int
+A = RegexFlag.A
```
Comment (AlexWaygood) [inline stubs/regex/regex/_regex_core.pyi:47]: Should all of the module constants in this file be `Final`?

```suggestion
A: Final = RegexFlag.A
```

Comment (hamdanal) [reply stubs/regex/regex/_regex_core.pyi]: Good idea, done.

Comment (hamdanal) [reply stubs/regex/regex/_regex_core.pyi]: Looks like pytype isn't happy about this change

**Review by srittau — APPROVED** [review]

File: stubs/regex/regex/_regex_core.pyi (hunk 1 above)
Comment (AlexWaygood) [reply stubs/regex/regex/_regex_core.pyi:47]: Ah well, thanks for trying! :-)

**Pull request merged**